study.md

```diff
@@ -67,5 +67,11 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+1. Client sends requests to server
+2. Server sends requests to controller
+3. Controller sends requests to model, parse requests into JSON
+4. Model looks into database
+5. Model sends response back to Controller
+6. Controller send response back to Server, parse responds into JSON
```

Comment:
Controllers render responses to the client and they don't necessarily need to be JSON.